src/plopp/__init__.py

```diff
@@ -16,7 +16,7 @@
 from .graph import show_graph
 from .plot import Plot
 from .model import Node, node, input_node
-from .figure import Figure
+from .figure import figure
```

Comment:
Will function name matching file name cause trouble?

Reply:
Good point. I don't remember in which case it does and doesn't.
For the sake of safety, I will change the file name.